tests/integration/bond_test.py

```diff
@@ -98,6 +99,23 @@ def bond99_with_slave(eth2_up):
         yield state
 
 
+@pytest.fixture
+def linux_br0_over_vlan101(bond99_with_slave):
```

Comment:
There are a few problems here:
The test attempts to define a bridge with a port that is a vlan that its base interface is a bond.  Providing a given bond using a fixture is fine, but all the rest makes less sense to do in the fixture because it is part of the test and not the setup.